config.sample.yaml

```diff
@@ -439,4 +439,19 @@ ircService:
   # Config for Matrix -> IRC bridging
   matrixHandler:
     # Cache this many matrix events in memory to be used for m.relates_to messages (usually replies).
-    eventCacheSize: 4096
\ No newline at end of file
+    eventCacheSize: 4096
+
+  ircHandler:
+    # How many /leave requests can be ongoing at a time.
+    # This is used to stem the flow of requests in case of a mass quit/leave, which might
+    # slow down the homeserver.
+    leaveConcurrency: 10
```

Comment:
no mention in the changelog?

Reply:
Good catch, this is a develop feature atm.